packages/grafana-ui/src/components/uPlot/config/UPlotConfigBuilder.ts

```diff
@@ -249,7 +249,7 @@ export class UPlotConfigBuilder {
   private tzDate = (ts: number) => {
     let date = new Date(ts);
 
-    return this.tz ? uPlot.tzDate(date, this.tz) : date;
+    return this.tz ? (require('uplot').default as typeof uPlot).tzDate(date, this.tz) : date;
```

Comment:
Not sure about this, or the pattern of moving all the uplot imports to `require` tbh.

Personally:
 - I don't like the move for _some_ imports being `import from`, and others being `require()` - it seems pretty inscrutable, and very easy for someone else to come along, move it to `import`, and break it again!
 - We don't want to introduce any more `as` typescript type assertions - we're a step away from introducing a lint rule to prevent them

Although I understand uplot doesn't make sense to use on the server, I am really keen to know what changes we can make to uplot to at least make it not blow up on _import_ @leeoniya

Reply:
required changes (https://github.com/leeoniya/uPlot/commit/01f716bbe42935bc2cb452a7d4918199bc90c210) are now in uPlot `master`.

will aim to cut a new release and bump dep next week.